src/components/data-management/LaunchAnalysisButton.jsx

```diff
@@ -99,29 +98,9 @@ const LaunchAnalysisButton = () => {
 
     const metadataKeysAvailable = activeExperiment.metadataKeys.length;
 
-    const allSampleFilesUploaded = (sample) => {
-      // Check if all files for a given tech has been uploaded
-      const { fileNames } = sample;
-      if (
-        !fileUploadSpecifications[sample.type].requiredFiles.every(
-          (file) => fileNames.includes(file.key),
-        )
-      ) { return false; }
```

Comment:
This check is unnecessary. If not all the required files are added then the sample can't even be created